acm/registration/near-edge/kustomization.yaml

```diff
@@ -2,5 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
+- openshift-gitops-managedclustersetbinding.yaml
+- openshift-gitops-placement.yaml
+- openshift-gitops-gitopscluster.yaml
```

Comment:
These 3 resources are not really _per application_: they're more just needed as prerequisites needed before any application.